tests/foreman/ui/test_contentview.py

```diff
@@ -17,6 +17,71 @@
 
 from robottelo.constants import FAKE_FILE_NEW_NAME, REPOS, DataFile
 
+VERSION = 'Version 1.0'
+
+
+@pytest.mark.tier2
+@pytest.mark.upgrade
+def test_positive_end_to_end(session, module_target_sat, module_org, module_lce, target_sat):
+    """Create content view with yum repo, publish it and promote it to Library
+
+    :id: 0970db8d-e9e3-4317-8703-153a6d9b2875
+
+    :steps:
+        1. Create Product/repo and Sync it
+        2. Create CV and add created repo in step1
+        3. Publish and promote it to new LCE
+
+    :expectedresults: content view is created, updated with repo publish and
+        promoted to  the LCE
+
+    :CaseImportance: High
```

Comment:
```suggestion
```
We can omit the importance here and "inherit" the module-wide from L11. Or set it to `Critical`. :thinking: